data.sql

```diff
@@ -12,4 +12,22 @@ INSERT INTO animals(name,date_of_birth,weight_kg,neutered,escape_attempts) VALUE
 INSERT INTO animals(name,date_of_birth,weight_kg,neutered,escape_attempts) VALUES ('Angemon','Jun-12-2005',45,TRUE,1);
 INSERT INTO animals(name,date_of_birth,weight_kg,neutered,escape_attempts) VALUES ('Boarmon','Jun-7-2005',20.4,TRUE,7);
 INSERT INTO animals(name,date_of_birth,weight_kg,neutered,escape_attempts) VALUES ('Blossom','Oct-13-1998',17,TRUE,3);
-INSERT INTO animals(name,date_of_birth,weight_kg,neutered,escape_attempts) VALUES ('Ditto','May-14-2022',22,TRUE,4);
\ No newline at end of file
+INSERT INTO animals(name,date_of_birth,weight_kg,neutered,escape_attempts) VALUES ('Ditto','May-14-2022',22,TRUE,4);
+
+/* Project 3 */
+
+INSERT INTO owners (full_name,age) VALUES ('Sam Smith', 34);
+INSERT INTO owners (full_name,age) VALUES ('Jennifer Orwell', 19);
+INSERT INTO owners (full_name,age) VALUES ('Bob', 45);
+INSERT INTO owners (full_name,age) VALUES ('Melody Pond', 77);
+INSERT INTO owners (full_name,age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners (full_name,age) VALUES ('Jodie Whittaker', 38);
+
+UPDATE animals SET species_id=2 WHERE name LIKE '%mon';
```

Comment:
- [ ] In between these two, you're missing a major insertion based on project requirements.
- [ ] Insertion the following data into the species table:
`Pokemon
Digimon`

## Requirement:
![image](https://github.com/muhammedb8826/vet-clinic-database/assets/50990925/d61b438b-3c9f-4e1c-8ba4-576ab20b1804)

Reply:
Hi Shedrack, Thank you for the feedback. I will add that to my data.sql file. However, I did that on the cmd. 😃